source/CellMeasurer/CellMeasurer.js

```diff
@@ -1,7 +1,7 @@
 /** @flow */
 import * as React from 'react';
 import {findDOMNode} from 'react-dom';
-import CellMeasurerCache from './CellMeasurerCache.js';
+import {CellMeasureCache} from './CellMeasurerCache.js';
```

Comment:
This won't actually change the generated prop-types:
```js
  cache:
    typeof _CellMeasurerCache.CellMeasureCache === "function"
      ? require("prop-types").instanceOf(_CellMeasurerCache.CellMeasureCache)
          .isRequired
      : require("prop-types").any.isRequired
```

Reply:
Made a few changes and the generated prop types now look like:
Will test it against react-virtualized-tree soon.

```
var babelPluginFlowReactPropTypes_proptype_CellMeasureCache = require('./types').babelPluginFlowReactPropTypes_proptype_CellMeasureCache || require('prop-types').any;

// ...
cache:
  typeof babelPluginFlowReactPropTypes_proptype_CellMeasureCache === 'function' 
    ? babelPluginFlowReactPropTypes_proptype_CellMeasureCache.isRequired 
      ? babelPluginFlowReactPropTypes_proptype_CellMeasureCache.isRequired 
      : babelPluginFlowReactPropTypes_proptype_CellMeasureCache 
    : require('proptypes').shape(babelPluginFlowReactPropTypes_proptype_CellMeasureCache).isRequired

```

Reply:
@bvaughn our current version of **babel-plugin-flow-react-proptypes** is far behind (^13.0.0) when the package is on 21.0.0.

Interface support was added in 17.0.0 (https://github.com/brigand/babel-plugin-flow-react-proptypes/pull/175), with the current version the propType for the interface is not being generated, causing the generated **types.js** to be empty. 

Is there any know blocker in updating the package or can I go for that?

Reply:
I upgraded it some time ago and published 5 releases trying to fix its issues. We can try again.

Reply:
Thanks @TrySound, was checking the release history and can see we tried to update to 15.0.0 at the time, were our issues caused by this bug (https://github.com/brigand/babel-plugin-flow-react-proptypes/commit/890c166ebc0f4db6df05fe6e58bebb322fccaae3) fixed in 16.0.0 release?

Reply:
will probably just cherry pick your original commit and change the version to the latest

Reply:
There was a few issues. I guess they are fixed now. I'm gonna land it after rollup integration.

Reply:
I have good news, tried to upgrade and all seems good on rv side, the build output seems to be converting flow types to propTypes correctly.

Will use a local npm link to test it against https://github.com/diogofcunha/react-virtualized-tree. If it works there I guess it's good to go.

Reply:
Tested this and after adding useESModules flag everything seems to be working fine for babel-plugin-flow-react-proptypes update, no tests were broken and I can't see any runtime errors.

Tomorrow will try to add a custom cache just to make sure the original issue gets resolved.